sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/builder/ConfigurationBuilders.java

```diff
@@ -1,59 +0,0 @@
-package org.sdase.commons.server.testing.builder;
-
-import io.dropwizard.core.Configuration;
-import java.util.function.Consumer;
-
-public interface ConfigurationBuilders {
```

Comment:
I agree that we can remove the interface without deprecating in `master`. 👍 
* it was only used by the already deprecated class `DropwizardConfigurationHelper`
* it's located within the `builders` package which already narrows the scope of the class
* [searching for it](https://github.com/search?q=org%3ASDA-SE%20ConfigurationBuilders&type=code) does not show any other usages in our services